h2o-core/src/main/java/water/api/FlowLoadServlet.java

```diff
@@ -0,0 +1,79 @@
+package water.api;
+
+import water.H2O;
+import water.JettyHTTPD;
+import water.persist.PersistManager;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.InputStream;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class FlowLoadServlet extends HttpServlet {
+
+  @Override
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
@pcejrowski Hmm, this doesn't actually handle any binary input/output. Seems to me it could be implemented as a Handler with much less code. Am I missing something?